libs/perun/components/src/lib/attributes-list/attribute-value/attribute-value-map/attribute-value-map.component.html

```diff
@@ -10,7 +10,8 @@
                 matInput
                 required
                 type="text"
-                (click)="readonly && attribute.displayName !== undefined && showValue(keys[i], attribute.displayName)"
+                class="overflow-ellipsis"
+                (click)="readonly && showValue(keys[i], attribute.displayName)"
```

Comment:
previous code: `(click)="readonly && attribute.displayName !== undefined && showValue(keys[i], attribute.displayName)"
` wasnt the attribute.displayName !== undefined important?